content/network-policies/tigera/tsce-sg-integration.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Integrating VPC Security Groups and Kubernetes Network Policy with TSCE"
+weight: 100
+---
+
+The network security that Calico provides in EKS is great, however it is primarily focused on the EKS cluster itself.  A common use-case for EKS, however, is to build a kubernetes cluster that can interact with other Amazon hosted resources, such as EC2 and RDS instances.  The native protection for those resources is the VPC's Security Group filtering.
+
+The problem with this, however, is that, by default, VPC Security Groups can only be applied to EC2 instances.  Therefore, if you wanted to allow some subset of your pods access to an RDS instance, for example, you would have to allow that access from all of your EKS worker nodes, thereby allowing ALL your EKS pods access to that RDS instance.  That's probably not what you want.  Luckily, one of the capabilities that TSCE enables is the integration of the VPC Security Group mechanism and Kubernetes/Calico network policy.
+
+Let's see how that works...
+
+# Create an EC2 resource in your VPC
+
+We're going to create a simple static webserver in your VPC, but not in your EKS cluster to act as a target to demonstrate how pods can become members of a VPC Security Group, and be referenced by VPC Security Group (VSG) policies.
+
+To do this, we need a simple web server in the same VPC as your EKS cluster, but running as a separate EC2 instance, not in the EKS cluster itself.  You can do this anyway you like, but if you want some hints, here are some steps you can take to accomplish this.
+
+* Go into your EC2 console and create an EC2 instances
+  * A t2-micro is more than sufficient
```

Comment:
Not a bad idea @brentley - but I am about to disappear for a while and will not have the cycles to get one crafted before I leave.  I agree though, it would be cleaner.